src/components/modals/confirmationDialog.test.ts

```diff
@@ -0,0 +1,19 @@
+import { mount } from '@vue/test-utils';
+import confirmationDialog from './confirmationDialog.vue';
+
+test('confirmationDialog', async () => {
+  const wrapper = mount(confirmationDialog, {
+    props: { show: true },
+    global: {
+      stubs: {
+        teleport: true,
+      },
+    },
+  });
+
+  await wrapper.get('#modal-confirm').trigger('click');
+  expect(wrapper.emitted()).toHaveProperty('result');
+  const result = wrapper.emitted('result')!;
+  expect(result).toHaveLength(1);
+  expect(result[0]).toStrictEqual([true]);
```

Comment:
you could also add a new test to check the correct handling of baseElement dismiss.